### PR TITLE
[config] Enable CUSE support. Contributes to JB#54227

### PR DESCRIPTION
**Pull request by HandsomeJaack**

### CODE REVIEW

**Comment by pvuorela** [comment]
@mkosola would you have any idea here?

File: mer_verify_kernel_config [hunk 1]
```diff
@@ -262,6 +262,7 @@ CONFIG_HIDRAW			y,m,!	# optional: Support HID devices
 CONFIG_UNIX			y	# UNIX sockets option is required to run Mer
 CONFIG_SYSVIPC			y	# Inter Process Communication option is required to run Mer
 CONFIG_EXT4_FS			y,m,!	# Mer uses ext4 as rootfs by default
+CONFIG_CUSE			y	# required for CUSE support.
```
Comment (mkosola) [inline mer_verify_kernel_config:265]: Should we have y,! here so it is recommended but not failing build if it is missing as it is not directly required?

Comment (rinatdobr) [reply mer_verify_kernel_config:265]: No, we expect that all new kernels will have that feature.

Comment (mkosola) [reply mer_verify_kernel_config:265]: Yeah, but everybody does not have new enough kernel with CUSE config who is porting the device for Sailfish

Comment (Thaodan) [reply mer_verify_kernel_config:265]: I would say they have since its there since Linux 2.6.31:
https://cateee.net/lkddb/web-lkddb/CUSE.html

Comment (Thaodan) [reply mer_verify_kernel_config:265]: In any case you could require it only starting with a kernel version.

**Review by Thaodan — CHANGES_REQUESTED** [review]
Please describe for which feature it is needed. All CONFIGS have an explanation for what or why there are needed.

File: mer_verify_kernel_config (hunk 1 above)
Comment (HandsomeJaack) [reply mer_verify_kernel_config]: Fixed

**Review by mkosola — APPROVED** [review]

**Review by Thaodan — CHANGES_REQUESTED** [review]
Why is this required for "software security modules support", what is that explain that more.
Why is it not optional? Currently none of the Sailfish X or community devices have this enabled.

**Comment by HandsomeJaack** [comment]
> Why is this required for "software security modules support", what is that explain that more. Why is it not optional? Currently none of the Sailfish X or community devices have this enabled.

Made optional. It's needed for OMP purposes.

**Pull request merged**